ui/cypress/plugins/index.js

```diff
@@ -27,5 +150,78 @@ module.exports = (on, config) => {
     );
   }
 
+  on('task', {
+
+    'debug': (...args) => {debug(...args); return true;},
+
+    // TODO this is unused for now as we can use API
+    // for simple cases, but for complex scenarios it will be useful
+    'seed': ({type, id, name}) => {
```

Comment:
My vision was to use `cy.task('seed'...)` everywhere on E2E tests so as to abstract away this chore, but didn't work as I wanted and is kinda a nice-to-have. Help needed. That is, change the test calls to seed from `cy.request(datasetUrl, data)... manual seed` to using `cy.task('seed', {type: 'dataset', ...)`